datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxUpgradeEndpoint.java

```diff
@@ -62,4 +64,19 @@ SdxUpgradeResponse upgradeClusterByCrn(@PathParam("crn") String crn,
     SdxCcmUpgradeResponse upgradeCcm(@ValidCrn(resource = CrnResourceDescriptor.ENVIRONMENT) @QueryParam("environment") @NotEmpty String environmentCrn,
             @ValidCrn(resource = { CrnResourceDescriptor.USER, CrnResourceDescriptor.MACHINE_USER })
             @NotEmpty @QueryParam("initiatorUserCrn") String initiatorUserCrn);
+
+    @PUT
+    @Path("{name}/upgrade_rds")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Upgrades the external database of the datalake", nickname = "upgradeSdxRdsByName")
```

Comment:
nit: external should be removed
also other endpoint nickname has datalake instead of sdx